styles/style.css

```diff
@@ -102,27 +96,23 @@ figure {
   display: block;
   position: absolute;
   top: 0;
-  left 0;
-
+  left: 0;
   width: 100%;
   height: 100%;
-
-  background: url('https://images.unsplash.com/44/C3EWdWzT8imxs0fKeKoC_blackforrest.JPG') no-repeat center center fixed;
+  background: url('https://images.unsplash.com/44/C3EWdWzT8imxs0fKeKoC_blackforrest.JPG' no-repeat center center fixed);
```

Comment:
Watch out here, the `)` matches the url so should come right after the end of the string.